v1/examples/pipeline/steps/custom/harness-approval-step.yaml

```diff
@@ -0,0 +1,31 @@
+version: 1
+kind: pipeline
+spec:
+  stages:
+    - type: approval
+      name: approval
+      id: Approval
+      timeout: 10m
+      when: <+<+environment.name> != "QA">
+      spec: 
+        steps:
+          - type: harness-approval
+            spec: 
+              approvers: 
+                min_count: 1
+                user_groups: 
+                  - account._account_all_users
+                disallow_executor: false
```

Comment:
Executor doesn't tell us what the behaviour is gonna be when someone selects true or false.
For instance when one chooses true, the behaviour is that the executor cannot approve/reject an approval
This could cause confusion.
cc: @hextechpal

Reply:
It's already under an approver sections, which implies that the executor is an approver or not

Reply:
@hextechpal can we connect once?

Reply:
https://harness.atlassian.net/wiki/spaces/~5f1f0648e8c4560022265233/pages/21100134856/RFC-005+-+Yaml+Syntax+Simplification#Booleans
@hextechpal we were following this doc, so should we keep this name as it is?